packages/descriptions/src/index.vue

```diff
@@ -96,7 +73,7 @@ export default defineComponent({
       return node
     }
 
-    const rows = computed(() => {
+    const getRows = () => {
```

Comment:
We can have a easier fix: just expose `getRows` and invoke it in template

Reply:
You are right, the reason I do this is because I think that using render function is more flexible

Reply:
Sadly render function is much harder for even component library authors. Avoid unless necessary.

Reply:
Okay, I see. I'll fix it